crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/fstring.py

```diff
@@ -289,6 +293,13 @@
          # comment 21
          }"
 
+x = f"{
+    x!s:>{
+        0
+        # comment 21
```

Comment:
I think 21 is already taken by the previous comment, let's choose another number so that it's easier to differentiate.